azure-pipelines.yml

```diff
@@ -30,7 +30,7 @@ stages:
 
       dependencies:
         - source: '\\nirvana\Measurements\VeriStandAddons\messaging_library'
-          file: 'CDMessaging.lvlibp'
+          file: '$target\CDMessaging.lvlibp'
```

Comment:
Not going to work until https://github.com/ni/niveristand-custom-device-build-tools/pull/163 is submitted.

Reply:
merged